version/version.go

```diff
@@ -0,0 +1,9 @@
+package version
+
+const (
+	// Version defines the gophernotes version.
+	Version string = "1.0.0"
+
+	// ProtocolVersion defines the Jupyter protocol version.
+	ProtocolVersion string = "5.0"
```

Comment:
s/ProtocolVersion/JupyterProtocolVersion?

Reply:
I prefer to keep ProtocolVersion since it is copied from gophernotes.